Emby.Server.Implementations/LiveTv/EmbyTV/EmbyTV.cs

```diff
@@ -2394,79 +2386,74 @@ private async Task UpdateTimersForSeriesTimer(SeriesTimerInfo seriesTimer, bool
             var allTimers = GetTimersForSeries(seriesTimer)
                 .ToList();
 
-            var registration = await _liveTvManager.GetRegistrationInfo("seriesrecordings").ConfigureAwait(false);
 
             var enabledTimersForSeries = new List<TimerInfo>();
-
-            if (registration.IsValid)
+            foreach (var timer in allTimers)
             {
-                foreach (var timer in allTimers)
+                var existingTimer = _timerProvider.GetTimer(timer.Id);
+
+                if (existingTimer == null)
                 {
-                    var existingTimer = _timerProvider.GetTimer(timer.Id);
+                    existingTimer = string.IsNullOrWhiteSpace(timer.ProgramId)
+                        ? null
+                        : _timerProvider.GetTimerByProgramId(timer.ProgramId);
+                }
 
-                    if (existingTimer == null)
+                if (existingTimer == null)
+                {
+                    if (ShouldCancelTimerForSeriesTimer(seriesTimer, timer))
+                    {
+                        timer.Status = RecordingStatus.Cancelled;
+                    }
+                    else
                     {
-                        existingTimer = string.IsNullOrWhiteSpace(timer.ProgramId)
-                            ? null
-                            : _timerProvider.GetTimerByProgramId(timer.ProgramId);
+                        enabledTimersForSeries.Add(timer);
                     }
+                    _timerProvider.Add(timer);
 
-                    if (existingTimer == null)
+                    if (TimerCreated != null)
                     {
+                        TimerCreated(this, new GenericEventArgs<TimerInfo>(timer));
```

Comment:
Just oneline this nullcheck,  this is consistent with the rest of the code.
```suggestion
                        TimerCreated?.Invoke(this, new GenericEventArgs<TimerInfo>(timer));
```

Reply:
Suggestions on GH still have a ways to go (only changes a single line). Thanks for the suggestion though.